Content.Server/Silicons/Borgs/BorgSystem.Modules.cs

```diff
@@ -281,15 +281,13 @@ public bool CanInsertModule(EntityUid uid, EntityUid module, BorgChassisComponen
                 if (!TryComp<ItemBorgModuleComponent>(containedModuleUid, out var containedItemModuleComp))
                     continue;
 
-                for (int i = 0; i < itemModuleComp.Items.Count; i++)
+                if (containedItemModuleComp.Items.All(itemModuleComp.Items.Contains) &&
+                    containedItemModuleComp.Items.Count == itemModuleComp.Items.Count)
```

Comment:
Flip the check so that the count check is run first. That way it's slightly faster.